### ISSUE TITLE
Mod Handshake Support?

### ISSUE THREAD

**Issue by RealFX-Code** [VelocityPowered/CrossStitch#6]
So, My fabric server refuses to connect to anything while behind Velocity, Because fabric isn't getting information on the mods I've installed on my client, thus kicks me every time I try to connect.

Is there a way to either bypass this, OR a fix for this?

Thanks, RealFX

**Comment by CCF100** [comment]
Did you ever get this fixed?

**Comment by RealFX-Code** [comment]
> Did you ever get this fixed?

Yes, But that was by disabling the mod cheking if the client had the mod.
Not ideal, but works.

I’m sure that an official fix would still be better. Becuse I know of some mods that doesn’t allow you to check if the client has the mod, they would just not work at all.
